src/main/java/com/enterprise/fastfoodapplication/controller/FastFoodController.java

```diff
@@ -74,6 +66,24 @@ public ResponseEntity fetchAllFood(){
             /*TO DO Logging*/
         }
 
+    }
+    /**
+     * This is for search bar on the navigation bar.
+     * This happens when the user click "search".
+     *
+     * @return
+     * */
+    @GetMapping("/SearchFood")
+    public String searchFood(@RequestParam(value="searchTerm",required = false,defaultValue = "None") String searchTerm, Model model) {
+        try {
+            List<Food> allFood = foodService.getAllFoodItems(searchTerm);
+            model.addAttribute("allFood", allFood);
+            return "allFood";
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "error";
```

Comment:
Exception handling is always a good idea.

Recommendation: Use proper logging instead of printStackTrace().
- Logging has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.